LeaderboardBackend/Migrations/20240718212110_AddLeaderboardTimestamps.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+using NodaTime;
+
+#nullable disable
+
+namespace LeaderboardBackend.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddLeaderboardTimestamps : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Instant>(
+                name: "created_at",
+                table: "leaderboards",
+                type: "timestamp with time zone",
+                nullable: false,
+                defaultValue: NodaTime.Instant.FromUnixTimeTicks(0L));
+
+            migrationBuilder.AddColumn<Instant>(
+                name: "deleted_at",
+                table: "leaderboards",
+                type: "timestamp with time zone",
+                nullable: true);
+
+            migrationBuilder.AddColumn<Instant>(
+                name: "updated_at",
+                table: "leaderboards",
+                type: "timestamp with time zone",
```

Comment:
`created_at` is non-nullable so it needs to have a default. These default values are kind of "dumb" right now because I haven't written the code that would install a trigger to properly initialize them yet. That's coming in a later PR because this one was already so bloated that I didn't want to add any more code to it.